python/pynq/drivers/usb_wifi.py

```diff
@@ -71,7 +71,11 @@ def __init__(self):
             if m:
                 if m.group(2) != 'lo' and m.group(2) != 'eth0':
                     self.wifi_port = m.group(2)
-        print(self.wifi_port)
+        if(hasattr(self,'wifi_port')):
+            print(self.wifi_port)
```

Comment:
I think it is better to have `self.wifi_port` always defined, even if it is None. 

Also, maybe we need to define another parameter `verbose`, and make its default value `True`; you will only print message when verbose is `True`, otherwise we don't print out anything.
i.e. something similar to: 
```python
__init__(self, verbose=True)
```